src/Components/PresentationalComponents/CSAwRuleBox/NotVulnerableBox.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Text,
+    TextContent,
+    TextVariants,
+    SplitItem,
+    Split,
+    Card,
+    CardBody,
+    ExpandableSection
+} from '@patternfly/react-core';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import messages from '../../../Messages';
+import './CSAwRuleBox.scss';
+import NotVulnerableLabel from '../Snippets/NotVulnerableLabel';
+import { changeExposedSystemsParameters } from '../../../Store/Actions/Actions';
+import { ONLY_NON_VULNERABLE_SYSTEMS } from '../../../Helpers/constants';
+
+const NotVulnerableBox = ({ synopsis, notVulnerableSystemCount }) => {
+    const dispatch = useDispatch();
+    const intl = useIntl();
+
+    const handleExposedSystemFilter = () => {
+        dispatch(changeExposedSystemsParameters({ rule: ONLY_NON_VULNERABLE_SYSTEMS }));
+    };
+
+    return (
+        notVulnerableSystemCount > 0 &&
+        <Card className="card-box" ouiaId="not-vulnerable-box">
+            <ExpandableSection toggleText={
+                <Split>
+                    <SplitItem className="pf-u-mr-xl">
+                        <TextContent>
+                            <Text component={TextVariants.h4}>
+                                <NotVulnerableLabel className="pf-u-mr-sm" />
+                                {intl.formatMessage(messages.notVulnerableBoxTitle)}
+                            </Text>
+                        </TextContent>
+                    </SplitItem>
+                    <SplitItem id="filter-affected-systems-split">
+                        <TextContent>
+                            <Text
+                                id="filter-affected-systems"
+                                onClick={event => {
+                                    handleExposedSystemFilter();
+                                    event.stopPropagation();
+                                }}
+                                component={TextVariants.small}
+                            >
+                                <Link
+                                    to={`/cves/${synopsis}/`}
```

Comment:
```suggestion
                                    to={`/cves/${synopsis}`}
```

Reply:
Fixed, but I'm wondering: is this just visual or is there a change in functionality?